sdk/core/azure-core-amqp/samples/eventhub_token_writer_sample/eventhub_token_writer_sample.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#include <azure/core/amqp/connection.hpp>
+#include <azure/core/amqp/connection_string_credential.hpp>
+#include <azure/core/amqp/message_sender.hpp>
+#include <azure/core/credentials/credentials.hpp>
+#include <azure/core/internal/environment.hpp>
+#include <azure/identity/client_secret_credential.hpp>
+#include <azure/identity/environment_credential.hpp>
```

Comment:
Is this header needed?

Reply:
No, because environment_credential actually work with new-testresources.ps1.